yaml/lang/fr.yml

```diff
@@ -0,0 +1,430 @@
+ACTOR:
+  TypeCharacter: Personnage
+  TypeMinion: Sbire
+  TypeRival: Rival
+  TypeNemesis: Nemesis
```

Comment:
These don't seem to be translated.

Reply:
Rival is the same. A dont put the accent on Nemesis a correct that